src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -860,4 +860,24 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase() {
             self::foo()
         }
     """)
+
+    fun `testE0199 Only safe impls for safe traits`() = checkErrors("""
```

Comment:
Annotation tests in this file are ordered respectively to their error codes. Would you, please, move them to the right positions? It makes maintenance a bit easier.